language-service/language_service/controller/__init__.py

```diff
@@ -0,0 +1,120 @@
+"""
+This handles the REST logic for communicating with clients
+"""
+
+import os
+import traceback
+
+from flask import request
+from flask_restful import Resource
+
+from service.nlp import tag
+from service.vocab import get_definitions
+
+AUTH_TOKEN = os.getenv("AUTH_TOKEN")
+SUPPORTED_LANGUAGES = ["CHINESE", "ENGLISH", "SPANISH"]
+
+
+def is_authorized(request):
+    return (
+        "Authorization" in request.headers
+        and request.headers["Authorization"] == AUTH_TOKEN
+    )
+
+
+class DefinitionController(Resource):
+    def get(self, language=None, word=None):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)